decidim-elections/app/services/decidim/votings/census_vote_flow.rb

```diff
@@ -53,9 +53,7 @@ def ballot_style_id
       private
 
       def ballot_style
-        return @ballot_style if defined?(@ballot_style)
-
-        @ballot_style = datum&.ballot_style
+        @ballot_style ||= datum&.ballot_style
```

Comment:
💬 This is a very minor thing, but I think that when dealing with `nil` or `false` values it's better to use the other approach. In this case, when the datum or the ballot_style is `nil`, the code will be executed every time. 😺

Reply:
Oh! I was actually wondering what was the difference between the two approaches!
Now I see :) ...reverting...!